src/openrct2-ui/drawing/engines/opengl/DrawCommands.h

```diff
@@ -27,35 +27,37 @@ template<typename T> class CommandBatch
         : _numInstances(0)
     {
     }
-    [[nodiscard]] bool empty() const
+
+    // NOLINTBEGIN
```

Comment:
Only ignore the `readability-identifier-naming` check, not all:
```suggestion
    // NOLINTBEGIN(readability-identifier-naming)
```

Reply:
Not sure if this already exists in the version we use. This seems to be a relatively new feature.

Reply:
Yeah it didn't work for me so i had to add the NOLINT comments as well